### ISSUE TITLE
Live TV Won't Play

### ISSUE THREAD

**Issue by dcbell** [jellyfin-archive/jellyfin-roku-legacy#1925]
## Software Versions

- Jellyfin Server Version:  10.9.11 (docker)
- Roku Client Version:  2.1 build 4

## Describe the bug
Movies and locally stored shows play fine. But when opening Live TV, after picking a channel it hangs for a while then displays a message so quick I can't read it before dropping back to the home screen.

Logs seem to indicate the server thinks things are streaming fine.

## How To Reproduce
Pick any channel in my live tv setup

## Logs
jellyfin logs:
```
[17:52:20] [INF] [31] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for [me]. EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[17:52:20] [INF] [31] Jellyfin.LiveTv.LiveTvMediaSourceProvider: Opening channel stream from Emby, external channel Id: m3u_341db0b31a53dd976428f7a819e86f16f40902c2f457d59c7795b16bb7668c20
[17:52:20] [INF] [31] Jellyfin.LiveTv.DefaultLiveTvService: Streaming Channel m3u_341db0b31a53dd976428f7a819e86f16f40902c2f457d59c7795b16bb7668c20
[17:52:20] [WRN] [34] Jellyfin.LiveTv.TunerHosts.M3UTunerHost: HEAD request to check MIME type failed, shared stream disabled
[17:52:20] [INF] [34] Jellyfin.LiveTv.TunerHosts.M3UTunerHost: Live stream opened after 0.1385ms
[17:52:20] [INF] [34] Jellyfin.LiveTv.DefaultLiveTvService: Returning mediasource streamId f40902c2f457d59c7795b16bb7668c20, mediaSource.Id f40902c2f457d59c7795b16bb7668c20, mediaSource.LiveStreamId null
[17:52:20] [INF] [35] Emby.Server.Implementations.Library.MediaSourceManager: Live tv media info probe took 0.0211102 seconds
[17:52:20] [INF] [35] Emby.Server.Implementations.Library.MediaSourceManager: Live stream opened: {"Protocol": "Http", "Id": "f40902c2f457d59c7795b16bb7668c20", "Path": "http://[url for media stream]", "EncoderPath": null, "EncoderProtocol": null, "Type": "Default", "Container": "ts", "Size": null, "Name": null, "IsRemote": true, "ETag": null, "RunTimeTicks": null, "ReadAtNativeFramerate": false, "IgnoreDts": true, "IgnoreIndex": false, "GenPtsInput": false, "SupportsTranscoding": true, "SupportsDirectStream": true, "SupportsDirectPlay": true, "IsInfiniteStream": true, "RequiresOpening": true, "OpenToken": null, "RequiresClosing": true, "LiveStreamId": "e2329f4997b378e64ccf8fa396deb76e_af999c25a00715699361240d4c6c7a53_f40902c2f457d59c7795b16bb7668c20", "BufferMs": null, "RequiresLooping": false, "SupportsProbing": true, "VideoType": null, "IsoType": null, "Video3DFormat": null, "MediaStreams": [{"Codec": "h264", "CodecTag": null, "Language": null, "ColorRange": null, "ColorSpace": null, "ColorTransfer": null, "ColorPrimaries": null, "DvVersionMajor": null, "DvVersionMinor": null, "DvProfile": null, "DvLevel": null, "RpuPresentFlag": null, "ElPresentFlag": null, "BlPresentFlag": null, "DvBlSignalCompatibilityId": null, "Comment": null, "TimeBase": "1/90000", "CodecTimeBase": null, "Title": null, "VideoRange": "SDR", "VideoRangeType": "SDR", "VideoDoViTitle": null, "AudioSpatialFormat": "None", "LocalizedUndefined": null, "LocalizedDefault": null, "LocalizedForced": null, "LocalizedExternal": null, "LocalizedHearingImpaired": null, "DisplayTitle": "720p H264 SDR", "NalLengthSize": "0", "IsInterlaced": false, "IsAVC": null, "ChannelLayout": null, "BitRate": 8000000, "BitDepth": 8, "RefFrames": 1, "PacketLength": null, "Channels": null, "SampleRate": null, "IsDefault": false, "IsForced": false, "IsHearingImpaired": false, "Height": 720, "Width": 1280, "AverageFrameRate": 59.94006, "RealFrameRate": 59.94006, "Profile": "High", "Type": "Video", "AspectRatio": "16:9", "Index": -1, "Score": null, "IsExternal": false, "DeliveryMethod": null, "DeliveryUrl": null, "IsExternalUrl": null, "IsTextSubtitleStream": false, "SupportsExternalStream": false, "Path": null, "PixelFormat": "yuv420p", "Level": 40, "IsAnamorphic": false, "$type": "MediaStream"}, {"Codec": "aac", "CodecTag": null, "Language": null, "ColorRange": null, "ColorSpace": null, "ColorTransfer": null, "ColorPrimaries": null, "DvVersionMajor": null, "DvVersionMinor": null, "DvProfile": null, "DvLevel": null, "RpuPresentFlag": null, "ElPresentFlag": null, "BlPresentFlag": null, "DvBlSignalCompatibilityId": null, "Comment": null, "TimeBase": "1/90000", "CodecTimeBase": null, "Title": null, "VideoRange": "Unknown", "VideoRangeType": "Unknown", "VideoDoViTitle": null, "AudioSpatialFormat": "None", "LocalizedUndefined": null, "LocalizedDefault": "Default", "LocalizedForced": null, "LocalizedExternal": "External", "LocalizedHearingImpaired": null, "DisplayTitle": "AAC - Stereo", "NalLengthSize": null, "IsInterlaced": false, "IsAVC": false, "ChannelLayout": "stereo", "BitRate": 195283, "BitDepth": null, "RefFrames": null, "PacketLength": null, "Channels": 2, "SampleRate": 48000, "IsDefault": false, "IsForced": false, "IsHearingImpaired": false, "Height": null, "Width": null, "AverageFrameRate": null, "RealFrameRate": null, "Profile": "LC", "Type": "Audio", "AspectRatio": null, "Index": -1, "Score": null, "IsExternal": false, "DeliveryMethod": null, "DeliveryUrl": null, "IsExternalUrl": null, "IsTextSubtitleStream": false, "SupportsExternalStream": false, "Path": null, "PixelFormat": null, "Level": 0, "IsAnamorphic": null, "$type": "MediaStream"}], "MediaAttachments": [], "Formats": [], "Bitrate": 8195283, "Timestamp": null, "RequiredHttpHeaders": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"}, "TranscodingUrl": null, "TranscodingSubProtocol": "http", "TranscodingContainer": null, "AnalyzeDurationMs": 3000, "TranscodeReasons": "0", "DefaultAudioStreamIndex": null, "DefaultSubtitleStreamIndex": null, "VideoStream": {"Codec": "h264", "CodecTag": null, "Language": null, "ColorRange": null, "ColorSpace": null, "ColorTransfer": null, "ColorPrimaries": null, "DvVersionMajor": null, "DvVersionMinor": null, "DvProfile": null, "DvLevel": null, "RpuPresentFlag": null, "ElPresentFlag": null, "BlPresentFlag": null, "DvBlSignalCompatibilityId": null, "Comment": null, "TimeBase": "1/90000", "CodecTimeBase": null, "Title": null, "VideoRange": "SDR", "VideoRangeType": "SDR", "VideoDoViTitle": null, "AudioSpatialFormat": "None", "LocalizedUndefined": null, "LocalizedDefault": null, "LocalizedForced": null, "LocalizedExternal": null, "LocalizedHearingImpaired": null, "DisplayTitle": "720p H264 SDR", "NalLengthSize": "0", "IsInterlaced": false, "IsAVC": null, "ChannelLayout": null, "BitRate": 8000000, "BitDepth": 8, "RefFrames": 1, "PacketLength": null, "Channels": null, "SampleRate": null, "IsDefault": false, "IsForced": false, "IsHearingImpaired": false, "Height": 720, "Width": 1280, "AverageFrameRate": 59.94006, "RealFrameRate": 59.94006, "Profile": "High", "Type": "Video", "AspectRatio": "16:9", "Index": -1, "Score": null, "IsExternal": false, "DeliveryMethod": null, "DeliveryUrl": null, "IsExternalUrl": null, "IsTextSubtitleStream": false, "SupportsExternalStream": false, "Path": null, "PixelFormat": "yuv420p", "Level": 40, "IsAnamorphic": false, "$type": "MediaStream"}, "$type": "MediaSourceInfo"}
[17:52:20] [INF] [35] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for david. EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
```
In the Dashboard > Logs, there are no ffmpeg logs for this.

## Connection Information

- Is server local or remote? Local
- Is server connection HTTP or HTTPS? HTTPS

**Comment by dcbell** [comment]
Forgot to mention, it plays fine on my desktop.

**Comment by jimdogx** [comment]
@dcbell this is something that has started happening since the server was updated to remux Live TV (instead of transcoding).  At least that is the leading theory.

However, in the meantime, some have found success by turning on the "Limit Bitrate" setting on the Roku client ("*" => Settings => Playback => Bitrate).  Something around 7 Mbps or lower seems to work.

This isn't ideal, but until it can be resolved, it's the only workaround I know of.

**Comment by dcbell** [comment]
> @dcbell this is something that has started happening since the server was updated to remux Live TV (instead of transcoding). At least that is the leading theory.
> 
> However, in the meantime, some have found success by turning on the "Limit Bitrate" setting on the Roku client ("*" => Settings => Playback => Bitrate). Something around 7 Mbps or lower seems to work.
> 
> This isn't ideal, but until it can be resolved, it's the only workaround I know of.

OK, I'll give that a try. Thank you. Is there a master bug report this can link to? Or should I just go ahead and close it?

**Comment by jimdogx** [comment]
> OK, I'll give that a try. Thank you. Is there a master bug report this can link to? Or should I just go ahead and close it?

I've been investigating this one the last few nights here and there.  So I'd leave this one open. :+1:

**Comment by jimdogx** [comment]
This should be fixed in the next 2.1.5 release of the client.
